openedx/features/discounts/utils.py

```diff
@@ -87,7 +87,7 @@ def format_strikeout_price(user, course, base_price=None, check_for_discount=Tru
             )).format(
                 original_price=original_price,
                 formatted_discount_price=formatted_discount_price,
-                s_sr=HTML("<span class='sr'>"),
+                s_sr=HTML("<span class='sr-only'>"),
```

Comment:
Bootstrap's CSS only recognizes `sr-only`, but LMS supports both. So I figured switch to the one that works in both.